ballerina-tests/tests/60_client_send_error_in_client_bidi_streaming_service.bal

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/grpc;
+import ballerina/io;
+
+listener grpc:Listener ep60 = new (9160);
+
+int clientStreamingErrorCount = 0;
+int bidiStreamingErrorCount = 0;
+
+@grpc:ServiceDescriptor {
+    descriptor: ROOT_DESCRIPTOR_60_CLIENT_SEND_ERROR_IN_CLIENT_BIDI_STREAMING,
+    descMap: getDescriptorMap60ClientSendErrorInClientBidiStreaming()
+}
+service "ErrorSendService" on ep60 {
+    remote function sendErrorClientStreaming(stream<string, error?> clientStream) returns int {
+        error? e = clientStream.forEach(isolated function(string val) {
+            io:println(val);
+        });
+        if e is () {
+            return clientStreamingErrorCount;
+        } else {
+            io:println("Received a client error : " + e.message());
```

Comment:
shall we log API to print the message? Normally we don't `io:println` in server-side implementation

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/444/commits/f0cc1d136eb72b71d1a97a917281c0b1fa9cb9a1